mitodl/edx-platform
# Added instructor dashboard button to push edX grades to Canvas

Closes #166 

To test:

- Run through the Canvas setup steps described in https://github.com/mitodl/edx-platform/pull/169
- Add some users to your local edX instance that match the enrollee emails in the Canvas test course (https://mit.test.instructure.com/courses/155/users)
- With those users, answer some of the graded items in your edX course (assignments, quizzes, etc.)
- Visit the Canvas section of the instructor dashboard and click the "push grades" button
- Check the gradebook on Canvas (https://mit.test.instructure.com/courses/155/gradebook) to confirm that all the edX assignments exist on Canvas, and that the grades have been properly updated

<img width="635" alt="ss 2020-08-12 at 22 53 47 " src="https://user-images.githubusercontent.com/14932219/90092558-3d32e580-dcf7-11ea-9994-8a35e4204bed.png">
